docs/source/performance/performance_long_sequence.md

```diff
@@ -0,0 +1,155 @@
+# Long Sequence Performance
+
+## LLAMA2-7B (FP8)
+
+- The table below shows the pre-training performance of the LLAMA2-7B with CP (context parallelism) and compares it against the results without CP at various input sequence lengths. The detailed model-parallel configurations and the achieved performance are shown in the training results with CP. In non-CP training runs, we use the most performant model- and data-parallel configurations without CP given the memory capacity constraint of the H100 GPU system.
+
+  - Container: [NeMo24.03.01.framework](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo/tags)
+  - System: DGX-H100
+
+<style>
```

Comment:
A minor issue, this part could be seen with preview mode. Is this the right behavior?

Reply:
I just followed the format from the https://github.com/NVIDIA/NeMo/blob/main/docs/source/performance/performance_summary.md. Could you let me know what was the expected behavior?

Reply:
I think you need to use markdown format which should be like:
| Model         | #-GPUs | GBS  | MBS | Sequence Length| TP | PP | CP | VP | Tokens / sec / GPU | Model TFLOP / sec / GPU | ***Est. time to train in days (10T tokens, 1K GPUs)*** |
```
| -----         | ------ | ---  | --- | ---------------| -- | -- | -- | -- | ------------------ | ----------------------- | ------------------------------------------------------ |
| GPT3-5B       | 64     | 2048 | 4   | 2048           | 1  | 1  | 1  | 1  | 23406              | 765                     | ***5***                                                |
| GPT3-20B      | 64     | 256  | 2   | 2048           | 2  | 1  | 1  | 1  | 5851               | 750                     | ***19***                                               |
| GPT3-175B     | 128    | 256  | 1   | 2048           | 4  | 8  | 1  | 6  | 716                | 771                     | **158**                                                |
| GPT3-175B     | 512    | 2048 | 2   | 2048           | 4  | 8  | 1  | 6  | 825                | [888](https://mlcommons.org/benchmarks/training/)                     | **137**                                                |
| LLAMA2-7B     | 8      | 128  | 1   | 4096           | 1  | 1  | 1  | 1  | 16934              | 780                     | ***7***                                                | 
| LLAMA2-13B    | 16     | 128  | 1   | 4096           | 1  | 4  | 1  | 10 | 8715               | 760                     | ***13***                                               |
| LLAMA2-70B    | 64     | 128  | 1   | 4096           | 4  | 4  | 1  | 20 | 1728               | 768                     | ***65***                                               |
| Nemotron-8B   | 64     | 256  | 4   | 4096           | 2  | 1  | 1  | 1  | 12507              | 643                     | ***9***                                                |
| Nemotron-22B  | 64     | 256  | 2   | 4096           | 2  | 4  | 1  | 10 | 4312               | 562                     | ***26***                                               |
| Nemotron-340B | 128    | 32   | 1   | 4096           | 8  | 8  | 1  | 12 | 326                | 686                     | ***347***                                              |
| LLAMA3-8B     | 8      | 128  | 1   | 8192           | 1  | 1  | 2  | 1  | 12273              | 711                     | ***9***                                                |
| LLAMA3-70B    | 64     | 128  | 1   | 8192           | 4  | 4  | 2  | 5  | 1524               | 734                     | ***74***                                               |
```

Reply:
I was trying to write a table with merged columns like this, which is not supported.
![image](https://github.com/user-attachments/assets/ccc23d9f-a344-461f-a7ca-e28c7b78e89c)

It worked well in vscode markdown preview, and the other systems like https://markdownlivepreview.com/ 
Will it cause an error in the Nemo doc web page?

Reply:
I see, it could be a github preview issue then, if it work fine on other places. I will approve this then.